Makefile

```diff
@@ -28,7 +35,22 @@ generate:
 	    --grpc_python_out=shiny \
 	    protos/shiny.proto
 
-clean:
-	rm -fr $(GRPCIO_VIRTUALENV) $(GOOGLEAPIS_PROTOS_DIR)
+.PHONY: docker-build
+docker-build:
+	docker build -t gcr.io/$(GCLOUD_PROJECT)/shiny .
+
+.PHONY: docker-push
+docker-push: docker-build
+	gcloud docker push gcr.io/$(GCLOUD_PROJECT)/shiny
```

Comment:
Wouldn't you rather have

```
"gcr.io/$(GCLOUD_PROJECT)/shiny"
```

in case there is whitespace in the project? (Not that there would be, but whitespace would break your command)